final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,92 @@
+import arg_parser
+import feedparser
+import sys
+from urllib.request import Request, urlopen
+from urllib.error import URLError, HTTPError
+import dominate
+import html
+
+import news_parser
+import logger
+import get_cache
+import converter
+
+
+def main(channel, limit):
+    '''
+    1. cache news
+    2. create html or pdf document
+    3. or print in stdout news in json or normal format
+    '''
+    news_parser.cacheNews(args.source, channel)
+
+    if (args.tohtml or args.topdf):
```

Comment:
Here and below: parenthesis are redundant here. You may want to check your application with `pycodestyle` tool